_episodes/03-lists.md

```diff
@@ -38,20 +38,27 @@ odds are: [1, 3, 5, 7]
 ~~~
 {: .output}
 
-We select individual elements from lists by indexing them:
+We can access elements of a list using indices -- numbered positions of elements in the list.
+These positions are numbered starting at 0, so the first element has an index of 0.
 
 ~~~
-print('first and last:', odds[0], odds[-1])
+print('first element:', odd[0])
+print('last element:', odd[3])
+print('"-1" element:', odd[-1])
 ~~~
 {: .language-python}
 
 ~~~
-first and last: 1 7
+first element: 1
+last element: 7
+"-1" element: 7
 ~~~
 {: .output}
 
-and if we loop over a list,
-the loop variable is assigned elements one at a time:
+We already saw how to use indexing to count elements starting from the beginning of the list, but we can also use negative indices to count backwards from the end.
```

Comment:
We've just demonstrated the `-1` index, so I would suggest saying something like this (feel free to modify/adjust):

> Yes, we can use negative numbers as indices in Python. When we do so, index of `-1` gives us the last element in the list, `-2` -- second to last, and so forth...

Reply:
Done

Reply:
Thank you! Could you please also break this line so we don't exceed a 100 characters per line limit?